Tests/RevenueCatUITests/CustomerCenter/ManageSubscriptionsViewModelTests.swift

```diff
@@ -196,6 +196,139 @@ class ManageSubscriptionsViewModelTests: TestCase {
         expect(subscriptionInformation.productIdentifier) == productIdOne
     }
 
+    func testShouldShowEarliestExpiration_whenUserHasLifetimeAndSubscriptionsOneEntitlement() async throws {
```

Comment:
I think this test is ensuring that the lifetime purchase is shown, could the test be named this?

```suggestion
    func testShouldShowLifetime_whenUserHasLifetimeAndSubscriptionsOneEntitlement() async throws {
```